### DISCUSSION TITLE
Empty page for 404 error for edit mode ?

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#3472] [General]
Hello

I mis-copied an url to my outline instance, and the rendered page was just an empty page.
The funny thing is the share mode has a 404 page
so 
https://outline.domain.com/ItDoesNotExist => empty page
but
https://outline.domain.com/share/ItDoesNotExist => NotFound

is it a bug in my installation ?

**ps**: btw both of these pages return a status code 200 instead of 404.

Best

**Comment by tommoor** [comment]
Feels like a bug to me, `200` is expected though

**Reply by bmillemathias** [reply]
Ok, I've seen you've opened an issue for that
thanks
